pkg/cluster/kubernetes/resource/load.go

```diff
@@ -203,14 +203,14 @@ func ParseMultidoc(multidoc []byte, source string) (map[string]KubeManifest, err
 // loadFile attempts to load a file from the path supplied. If sopsEnabled is set,
 // it will try to decrypt it before returning the data
 func loadFile(path string, sopsEnabled bool) ([]byte, error) {
-	bytes, err := ioutil.ReadFile(path)
+	fileBytes, err := ioutil.ReadFile(path)
 	if err != nil {
 		return nil, err
 	}
-	if sopsEnabled {
-		return softDecrypt(bytes)
+	if sopsEnabled && bytes.Contains(fileBytes, []byte("sops:")) {
```

Comment:
unlikely to be triggered IRL, but this disables sops for JSON-like content of yaml files